api/utils.py

```diff
@@ -162,3 +168,43 @@ def extract_post_params(
             if value is not None:
                 data[param] = value
     return data
+
+
+def get_object_or_json_error(klass, *args, **kwargs):
+    """
+    This is exact replica of `django.shortcuts.get_object_or_404()` function
```

Comment:
Please include a permalink to the Django source as part of the docstring. You can add a "References" section toward the end with a bulleted list of links.

Reply:
Done.